public/css/custom.css

```diff
@@ -507,10 +507,18 @@ label[for=password] {
     font-size: 1.5rem;
 }
 
-/* Results for Live Search */
-.search-result:focus,
```

Comment:
Is there a reason this was removed?

Reply:
No, i don't remember taking that out.. must've done it accidentally. I'll fix that now

Reply:
Done.